src/main/java/com/dap/DailyArtPrompt/repository/UserRepository.java

```diff
@@ -1,10 +1,12 @@
 package com.dap.DailyArtPrompt.repository;
 
 import com.dap.DailyArtPrompt.entity.User;
+import com.dap.DailyArtPrompt.model.UserResponse;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
     User findByEmail(String email);
+    UserResponse findByEmailAndPassword(String email, String password);
```

Comment:
This should return me a `User`

Reply:
Actually I don't want it to return a User because then the password will be returned

Reply:
That's why I made UserResponse so I only return the id and email